### PR TITLE
[development-rules] Test commit. Closed #6

### LINKED ISSUE #6: Тестовый коммит еще один

### PR DESCRIPTION
**Pull request by RostovtsevDanila**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 Основные правила и гайды для разработчиков команды Elevons.
+
+test
```
Comment (atatarov) [inline README.md:3]: wtf

Comment (RostovtsevDanila) [reply README.md:3]: test))))0)

**Comment by atatarov** [comment]
hello

**Comment by RostovtsevDanila** [comment]
Hi

**Review by atatarov — APPROVED** [review]

**Pull request merged**